htdocs/core/lib/functions.lib.php

```diff
@@ -8771,7 +8771,19 @@ function complete_head_from_modules($conf, $langs, $object, &$head, &$h, $type,
 							complete_substitutions_array($substitutionarray, $langs, $object, array('needforkey'=>$values[2]));
 							$label = make_substitutions($reg[1], $substitutionarray);
 						} else {
-							$label = $langs->trans($values[2]);
+							$labeltemp = explode(',', $values[2]);
+							if (is_array($labeltemp)) {
+								$label = $langs->trans($labeltemp[0]);
+								if (!empty($labeltemp[1]) && is_object($object) && !empty($object->id)) {
+									dol_include_once($labeltemp[2]);
+									$obj = new $labeltemp[1]($db);
+									$function = $labeltemp[3];
+									$nbrec = $obj->$function($object->id, $obj);
```

Comment:
If $funtion is not set or does not exists, it will make a fatal error

Reply:
I resolv problem